rs/src/whitespace.rs

```diff
@@ -0,0 +1,12 @@
+use nom::{character::complete::multispace0, error::ParseError, sequence::delimited, IResult};
+
+/// chomp trailing whitespace
```

Comment:
I believe this should probably be:
```suggestion
/// chomp whitespace at either end of a combinator
```
or similar.